app/controllers/rockets_controller.rb

```diff
@@ -1,11 +1,19 @@
 class RocketsController < ApplicationController
+  before_action :set_rocket, only: [:show, :edit, :update, :destroy]
   def new
+    @rocket = Rocket.new
   end
 
   def index
+    @rockets = Rocket.all
   end
 
   def create
+    @rocket = Rocket.new(rocket_params)
+    @rocket.save
```

Comment:
what about the 
if @rocket.save
redirect_to ...
else render: new
end

Reply:
Done